README.md

```diff
@@ -23,52 +23,52 @@ Official website: https://parity.io | Be sure to check out [our wiki](https://wi
 
 ----
 
-## About Parity
+## About Parity-Ethereum
 
-Parity's goal is to be the fastest, lightest, and most secure Ethereum client. We are developing Parity using the sophisticated and cutting-edge Rust programming language. Parity is licensed under the GPLv3, and can be used for all your Ethereum needs.
+Parity-Ethereum's goal is to be the fastest, lightest, and most secure Ethereum client. We are developing Parity-Ethereum using the sophisticated and cutting-edge Rust programming language. Parity-Ethereum is licensed under the GPLv3, and can be used for all your Ethereum needs.
 
-From Parity Ethereum client version 1.10.0, the User Interface (UI) is accessible in a separate application called Parity UI. To keep using the UI in the browser (deprecated), [follow these steps](https://wiki.parity.io/FAQ-Basic-Operations,-Configuration,-and-Synchronization#the-parity-ui-application-isnt-working-the-way-i-want).
+From Parity-Ethereum client version 1.10.0, the User Interface (UI) is accessible in a separate application called Parity-UI. To keep using the UI in the browser (deprecated), [follow these steps](https://wiki.parity.io/FAQ-Basic-Operations,-Configuration,-and-Synchronization#the-parity-ui-application-isnt-working-the-way-i-want).
```

Comment:
I'd change this to "The UI for Parity-Ethereum is available in a separate application called … …" – not sure why anyone reading this would care about when the UI was extracted?